src/Tasks/Common/Resources/Strings.resx

```diff
@@ -538,4 +538,16 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1104: RollForward value '{0}' is invalid. Allowed values are {1}.</value>
     <comment>{StrBegin="NETSDK1104: "}</comment>
   </data>
+  <data name="WindowsDesktopFrameworkRequiresVersion30" xml:space="preserve">
```

Comment:
Is this going to be used by a future change?

Reply:
Yes, as soon as this change flows into `dotnet/wpf`, we will consume this in `Microsoft.NET.Sdk.WindowsDesktop`